libtenzir/src/curl.cpp

```diff
@@ -125,15 +125,30 @@ auto easy::set_postfieldsize(long size) -> code {
   return static_cast<code>(curl_code);
 }
 
+auto easy::set_cacert(std::string_view path) -> code {
+  auto curl_code = curl_easy_setopt(easy_.get(), CURLOPT_CAINFO, path.data());
+  return static_cast<code>(curl_code);
+}
+
+auto easy::set_certfile(std::string_view path) -> code {
+  auto curl_code = curl_easy_setopt(easy_.get(), CURLOPT_SSLCERT, path.data());
+  return static_cast<code>(curl_code);
+}
+
+auto easy::set_keyfile(std::string_view path) -> code {
+  auto curl_code = curl_easy_setopt(easy_.get(), CURLOPT_SSLKEY, path.data());
+  return static_cast<code>(curl_code);
+}
+
```

Comment:
I don't think we need these. We have one function for setting options already:

```
auto easy::set(CURLoption option, long parameter) -> code {
  auto curl_code = curl_easy_setopt(easy_.get(), option, parameter);
  return static_cast<code>(curl_code);
}
```

We only create special-purpose function, we need types as arguments that go beyond string views, longs, etc..

Reply:
Please remove the functions and replace them with calls like this:

```cpp
easy::set(CURLOPT_CAINFO, path);
```